### PR TITLE
updates to chef-server-ctl gather-logs

### PR DESCRIPTION
**Pull request by moutons**
-added AWS to known platforms
-added AWS Native Chef Server info
-added Elasticsearch info
-switched compression from bzip2 to gzip

Signed-off-by: Shaun Mouton <smouton@chef.io>

### Description

updates `chef-server-ctl gather-logs` with more useful info for troubleshooting

### Issues Resolved

adds error message to address #1463 

### Check List

- [ ] New functionality includes tests
- [ ] All buildkite tests pass
- [ ] Full omnibus build and tests in buildkite pass
- [ ] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>
- [ ] PR title is a worthy inclusion in the CHANGELOG

### CODE REVIEW

File: omnibus/files/private-chef-scripts/gather-logs [hunk 1]
```diff
@@ -99,6 +107,30 @@ for i in `find -L /var/log/${path}* -type f -mmin -"$modified_within_last_x_minu
...
+    curl -s -X GET "http://localhost:$1/_cat/shards?h=index,shard,prirep,state,unassigned.reason&pretty" > "$tmpdir/elasticsearch-cat-shards.txt"
+    curl -s -X GET "http://localhost:$1/_cluster/allocation/explain?pretty" > "$tmpdir/elasticsearch-cluster-shards-explain.txt"
+}
+ES_9200="$(curl -s -XGET 'http://localhost:9200/' | grep 'for Search')"
+ES_10144="$(curl -s -XGET 'http://localhost:10144/' | grep 'for Search')"
+if [[ -n $ES_9200 ]];
+then
+    gather_es 9200
+elif [[ -n $ES_10144 ]];
+    gather_es 10144
```
Comment (PrajaktaPurohit) [inline omnibus/files/private-chef-scripts/gather-logs]: gather_es 10144 should be under the `then` case

File: omnibus/files/private-chef-scripts/gather-logs [hunk 2]
```diff
@@ -21,6 +21,14 @@ then
     testing=true
 fi
 
+#Don't continue if bzip isn't installed
```
Comment (markan) [inline omnibus/files/private-chef-scripts/gather-logs]: Do we not ship with bzip? Should bzip be a dependency in chef server omnibus/hab packages? Or, alternatively is there something (gzip) that we do ship? What's the performance cost of switching?

I'd love to figure out how to have a 'batteries included' system so this check isn't needed.

Comment (moutons) [reply omnibus/files/private-chef-scripts/gather-logs]: I'm guessing that compression (to meet zendesk's 20MB/file limit) was the reason to use bzip, and bzip2 is not a hard dependency of the package (whether debian or EL) right now.

Comment (moutons) [reply omnibus/files/private-chef-scripts/gather-logs]: switching to gzip and removing check

**Pull request merged**